docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1217,7 +1217,7 @@ expression that supports aliased expressions that can be reused.
 This tool is perfect for syslog logs, apache and other webserver logs, mysql logs, and in general, any log format
 that is generally written for humans and not computer consumption.
 This processor comes packaged with over
-https://github.com/elastic/elasticsearch/tree/master/modules/ingest-common/src/main/resources/patterns[120 reusable patterns].
+https://github.com/elastic/elasticsearch/blob/master/libs/grok/src/main/resources/patterns/grok-patterns[120 reusable patterns].
```

Comment:
Nit: a slight improvement would be to link to the grok-patterns file on the current branch (either master or one of the version branches, e.g. 6.x, 6.2). You can use the `{branch}` placeholder for this inside the link as I just learned [recently](https://github.com/elastic/elasticsearch/pull/28530#issuecomment-363725730)

Reply:
Thanks for your suggestion. It's fixed now.